HIRS_AttestationCAPortal/src/main/java/hirs/attestationca/portal/page/controllers/ReferenceManifestPageController.java

```diff
@@ -177,17 +187,44 @@ protected RedirectView upload(
             final RedirectAttributes attr) throws URISyntaxException, Exception {
         Map<String, Object> model = new HashMap<>();
         PageMessages messages = new PageMessages();
+        List<MultipartFile> rims = new ArrayList<>();
+        String fileName;
+        Path filePath;
 
         // loop through the files
         for (MultipartFile file : files) {
+            fileName = file.getOriginalFilename();
+            if (fileName.toLowerCase().endsWith("swidtag")) {
+                rims.add(file);
+            } else {
+                filePath = Paths.get(String.format("%s/%s",
+                            SwidResource.RESOURCE_UPLOAD_FOLDER,
+                            file.getOriginalFilename()));
+                if (Files.notExists(Paths.get(SwidResource.RESOURCE_UPLOAD_FOLDER))) {
```

Comment:
Should this check occur before the Paths.get(String....) call immediately preceding it?

Reply:
The Path variable isn't acting on the location provided, so if it doesn't exists, nothing bad happens.